larray/core/array.py

```diff
@@ -8816,15 +8875,17 @@ def stack(elements=None, axis=None, title=None, meta=None, dtype=None, **kwargs)
 
         Stacking sessions will return a new session containing the arrays of all sessions stacked together. An array
         missing in a session will be replaced by NaN.
-    axis : str or Axis or Group, optional
-        Axis to create. If None, defaults to a range() axis.
+    axes : str or Axis or Group or tuple/AxisCollection of Axis, optional
+        Axes to create. If None, defaults to a range() axis.
     title : str, optional
         Deprecated. See 'meta' below.
     meta : list of pairs or dict or OrderedDict or Metadata, optional
         Metadata (title, description, author, creation_date, ...) associated with the array.
         Keys must be strings. Values must be of type string, int, float, date, time or datetime.
     dtype : type, optional
         Output dtype. Defaults to None (inspect all output values to infer it automatically).
+    res_axes : AxisCollection, optional
```

Comment:
add an example below?

Reply:
I wouldn't because that's a very niche usage (that I implemented for internal use within LArray) and I don't think it's a good idea to always show all possible arguments in the examples. I should add a unit test for that though.

Reply:
OK

Reply:
done